### PR TITLE
val3_driver: Updated repo URL within README

### PR DESCRIPTION
**Pull request by MuriloMartins**
This PR addresses issue #13.

### CODE REVIEW

File: staubli_val3_driver/README.md [hunk 1]
```diff
@@ -25,10 +25,10 @@ contents of the `val3_driver` folder to the controller itself.
 
 ### Clone this repository
 
-Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ocadotechnology/staubli_experimental):
+Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ros-industrial/staubli_experimental):
 
 ```shell
-git clone https://github.com/ocadotechnology/staubli_experimental -b val3_server-devel
+git clone https://github.com/ros-industrial/staubli_experimental -b val3_server-devel
```
Comment (gavanderhoorn) [inline staubli_val3_driver/README.md]: Shouldn't users clone just `indigo-devel`? Afaik there is no `val3_server-devel` branch on `ros-industrial/staubli_experimental`?

And as `indigo-devel` is the default, I think we can just omit the branch entirely.

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Minor change needed, otherwise +1.

File: staubli_val3_driver/README.md (hunk 1 above)
Comment (MuriloMartins) [reply staubli_val3_driver/README.md]: I missed that... done.

File: staubli_val3_driver/README.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ contents of the `val3_driver` folder to the controller itself.
 
 ### Clone this repository
 
-Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ocadotechnology/staubli_experimental):
+Clone branch `val3_server-devel` of [staubli_experimental](https://github.com/ros-industrial/staubli_experimental):
```
Comment (gavanderhoorn) [inline staubli_val3_driver/README.md]: I think you missed this one :)

**Comment by gavanderhoorn** [comment]
Btw: the readme also directs users to:

> [..] copy the contents of `val3_driver` folder [..]

But that folder doesn't exist. The same `val3_driver` name is used further on in the document as well.

Should that be `val3`? Or would you want them to copy the `ros_libs` and `ros_server` directories separately?

**Comment by MuriloMartins** [comment]
Oh, I seem to remember to have named that folder as `val3_driver`. I didn't realise that it had been renamed...

**Comment by gavanderhoorn** [comment]
@MuriloMartins wrote:

> Oh, I seem to remember to have named that folder as `val3_driver`. I didn't realise that it had been renamed...

ah, right. Yes, I did that (when cleaning up the commits in #10) to get `staubli_val3_driver` more in-line with the other driver packages (ie: `fanuc_driver/karel`, `abb_driver/rapid`, etc). The directory already lives in a package with `driver` in it, so seems redundant to suffix it again.

**Comment by MuriloMartins** [comment]
Just a sec, I'm working on it! ;-)

**Comment by gavanderhoorn** [comment]
https://github.com/ros-industrial/staubli_experimental/pull/10#issuecomment-231539601 explains what happened :).

**Comment by MuriloMartins** [comment]
Yes I remember now! Thanks!

I've just updated the readme with the correct branch (`indigo-devel`) and folder (`val3`) names.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Thanks for the updates.

I'm going to squash-merge this.

**Pull request merged**